go-controller/pkg/ovn/base_network_controller_multipolicy.go

```diff
@@ -103,6 +113,7 @@ func convertMultiNetPolicyToNetPolicy(mpolicy *mnpapi.MultiNetworkPolicy, allowP
 			egress.Ports[j] = knet.NetworkPolicyPort{
 				Protocol: mport.Protocol,
 				Port:     mport.Port,
+				EndPort:  ptrIntToInt32(mport.EndPort),
```

Comment:
https://github.com/k8snetworkplumbingwg/multi-networkpolicy/pull/21 got merged

@cathy-zhou could you please bump github.com/k8snetworkplumbingwg/multi-networkpolicy to latest? The type conversion here is not required anymore.

CC @tssurya

Reply:
@l8huang @tssurya done. please take a look.

Reply:
/lgtm